vortex-serde/src/message_writer.rs

```diff
@@ -85,7 +85,7 @@ impl<W: VortexWrite> MessageWriter<W> {
         Ok(())
     }
 
-    async fn write_message<F: WriteFlatBuffer>(&mut self, flatbuffer: F) -> io::Result<()> {
+    pub async fn write_message<F: WriteFlatBuffer>(&mut self, flatbuffer: F) -> io::Result<()> {
```

Comment:
this struct is not exposed and this function is very useful

Reply:
This is now used in LayoutWriter to avoid duplicating the flatbuffer serialization logic